packages/playground/src/components/smtp_server.vue

```diff
@@ -46,7 +45,7 @@
           :rules="[
             validators.required('Password is required.'),
             validators.minLength('Password must be at least 6 characters.', 6),
-            validators.maxLength('Password cannot exceed 50 characters.', 50),
+            validators.maxLength('Password cannot exceed 69 characters.', 69),
```

Comment:
can you explain why 69 chars?

Reply:
Because we use ```sendgrid``` and its password is about 69 chars.